openff/toolkit/tests/test_molecule.py

```diff
@@ -3711,25 +3711,46 @@ def test_get_bond_between(self):
     @requires_rdkit
     def test_molecule_rings(self, smiles, n_rings):
         """Test the Molecule.rings property"""
-        assert (
-            n_rings == Molecule.from_smiles(smiles, allow_undefined_stereo=True).n_rings
+        mol = Molecule.from_smiles(smiles, allow_undefined_stereo=True)
+        assert n_rings == mol.get_n_rings()
+
+    def test_is_in_ring(self):
+        """
+        Test basic behavior of Atom.is_in_ring and Bond.is_in_ring API.
+        More extensive behavior testing is done in test_toolkits.py
+        """
+        molecule = Molecule.from_smiles(
+            "[H:9][c:3]1[c:2]([c:1]([c:6]([c:5]([c:4]1[H:10])[H:11])[H:12])[H:7])[H:8]"
         )
+        assert molecule.atoms[1].is_in_ring()
+        assert molecule.bonds[1].is_in_ring()
```

Comment:
(blocking) This is not appropriate - the SMILES above does not specify an order for the bonds, so it would be valid for a toolkit to say `bonds[1]` is a C-H bond.

Reply:
👍 I hadn't considered that bond order is not so trustworthy. Since this is a test in the Molecule API, it's intentionally simple. I rewrote it to check that every C-C bond in benzene is "in a ring"